RUFAS/routines/field/crop/crop.py

```diff
@@ -103,7 +103,7 @@ def biomass_allocation(self) -> BiomassAllocation:
         return self._biomass_allocation
 
     @property
-    def nitrogen_incorporation(self) -> NitrogenIncorporation:
+    def nitrogen_incorporation(self) -> NitrogenUptake:
         """Provides access to the NitrogenIncorporation object."""
```

Comment:
```suggestion
        """Provides access to the NitrogenUptake object."""
```
And also change the return if you end up changing the attribute name as well.